### PR TITLE
Add receive modal story.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/wallets/receive-modal/index.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        To receive Lumens or assets from non-Keybase users, pass your Stellar addresses around:
+      </Text>
+      <Box2
+        direction="vertical"
+        style={{
+          marginBottom: globalMargins.tiny,
+          width: '100%',
+        }}
+      >
+        <CopyText text="G23T5671ASCZZX09235678ASQ511U12O91AQ" />
```
Comment (akalin-keybase) [inline shared/wallets/receive-modal/index.js]: I think the idea is to make this component close to the real thing, i.e. have a prop for the Stellar address, and also the e-mail address below, and the keybase name above

Comment (adamjspooner) [reply shared/wallets/receive-modal/index.js]: I wasn't sure how close to the real thing these are supposed to be when we're implementing the storybook. I'll add those props. Thanks!

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for another review.

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

File: shared/wallets/receive-modal/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        type={isMobile ? 'icon-wallet-receive-64' : 'icon-wallet-receive-48'}
+        style={{marginBottom: globalMargins.small}}
+      />
+      <Text type="BodySmallSemibold">{props.username}’s wallet</Text>
+      <Text type="Header" style={{marginBottom: globalMargins.medium}}>
+        Receive
+      </Text>
+      <Text
+        type="Body"
+        style={{
```
Comment (chrisnojima) [inline shared/wallets/receive-modal/index.js]: why not have these styles in the stylesheet create below

Comment (adamjspooner) [reply shared/wallets/receive-modal/index.js]: I wasn't sure what the threshold was for moving styles into `styleSheetCreate`. Should we always use that, regardless of the number of styles? For instance, should the single margin declarations be moved too?

Comment (chrisnojima) [reply shared/wallets/receive-modal/index.js]: i think its a lot cleaner if they're all down there

Comment (adamjspooner) [reply shared/wallets/receive-modal/index.js]: Works for me. Thanks!

**Pull request merged**